### ISSUE TITLE
Migrate to Jakarta namespace

### ISSUE THREAD

**Issue by Thihup** [severell/severell#12]
As a result of Oracle transferring the Java EE to Eclipse Foundation, the namespace had to be changed.
Is a matter of renaming the packages from javax to jakarta.

https://jakarta.ee

**Comment by mitchdennett** [comment]
Yeah, I never really kept track of all that! I guess that is something we will need to change
